vignettes/Vaccines.Rmd

```diff
@@ -147,16 +147,16 @@ rtssepiparams <- simparams
 
 # Add RTS,S strategy
```

Comment:
Line 95 above - let's change the age group of the mass strategy to something more realistic (not 17 years). I remember this example confused Amir